Tools/WebKitTestRunner/WebKitTestRunner.xcodeproj/project.pbxproj

```diff
@@ -231,6 +250,18 @@
 			name = "Copy Plug-Ins";
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		E372BC342C08E22D006DFE67 /* CopyFiles */ = {
```

Comment:
Could you give this a name, maybe "Embed extensions"? (In Xcode, click on the "Copy Files" label and it will magically become a text field!)

Reply:
Yes, good point, will do!